less/index.less

```diff
@@ -49,11 +49,114 @@ table {
 }
 
 /* Set every element's box-sizing to border-box */
-* {
+*
```

Comment:
Not sure if this was intentional but it looks like it belongs in line 60 before {